### ISSUE TITLE
Rotation and translation initialization during training

### ISSUE THREAD

**Issue by findmoon1** [NVlabs/FoundationPose#350]
Your work is so good. I would like to ask if the initial rotation is randomly offset during training and the translation is set to the depth value, the range of translation that needs to be updated will be very small. When setting the loss, if the weight is set at 1:1, will it cause a large translation error?

**Comment by tetanus-sy** [comment]
@findmoon1 Hi，Have you successfully tried fine-tuning this network?

**Comment by minqqqqqq** [comment]
@findmoon1 您好，想问一下，您知道这个网络的训练代码吗？我也想微调这个网络，但是没有找到训练代码，或者您有如何训练的思路吗？
